concordium-consensus/package.yaml

```diff
@@ -267,6 +267,7 @@ tests:
     - hspec-expectations >= 0.8
     - vector >= 0.12
     - math-functions >= 0.2
+    - temporary >= 1.3
```

Comment:
Where is this used?

Reply:
In a test. I forgot to commit this previously (when migrating a consensus test) so the CI is failing now on main. So I just fixed it in this PR.